src/controllers/util/index.ts

```diff
@@ -1,15 +1,23 @@
 import { Context } from 'koa';
 import { API_VERSION } from '../../routes/utils/constants';
 import { isHttpStatusSuccess } from '../../v0/util';
+import { HttpStatus } from '../../v0/util/constant';
 
 export default class ControllerUtility {
+  private static handleForNotfoundStatus(status: number) {
+    if (status === HttpStatus.NOT_FOUND) {
+      return HttpStatus.BAD_REQUEST;
```

Comment:
To generalize, we can term it as a bad request but for the destination under discussion, shouldn't it be a configuration error?

Reply:
I meant this status code translation should happen within that destination (Slack). The existing one suffices for the generic handler.

Reply:
Yes, ideally we should handle in specific destination's handlers explicitly.